src/styles/index.css

```diff
@@ -0,0 +1,253 @@
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  height: 60px;
+  text-transform: uppercase;
+}
+
+.nav__element {
+  margin-left: 20px;
+}
+
+.nav__element:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  color: #060b35;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 60px;
+  display: inline-block;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  width: 200px;
+}
+
+.card__image {
+  margin-top: 16px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+}
+
+.card__title-text {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__title-product-code {
+  font-family: Roboto, serif;
+  font-weight: 400;
+  font-style: normal;
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+  margin-top: 4px;
+}
+
+.card:hover {
+  transform: scale(1.2);
+  transition: transform 300ms;
+}
+
+.card:hover .card__title-text {
+  color: #34568b;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 16px;
+  width: 100%;
+  margin-top: 16px;
+}
+
+.stars__list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  justify-content: space-between;
```

Comment:
The `justify-content` property is used here, but it seems the `.stars__star` class is not defined as a flex container. Ensure that `display: flex;` is applied to the `.stars__star` class if you intend to use flex properties.